src/components/SlideshowHero/_slideshow-hero.scss

```diff
@@ -194,10 +194,22 @@
 }
 
 .slideshow__image {
-  height: 100%;
-  object-fit: cover;
   position: absolute;
   width: 100%;
+
+  @supports (object-fit: cover) {
+    height: 100%;
+    object-fit: cover;
+  }
+
+  @media all and (-ms-high-contrast:none) {
+    top: 50%;
+    transform: translateY(-50%);
+  }
+
+  &.lazy {
```

Comment:
Wondering about this classname... thinking should it be:

- `is-lazy` to denote a state... but it's not stateful really... so should it be
- `.slideshow-image--lazy` to denote modifier?

Reply:
`.u-lazy` prob better since (I hope) it'll become universal

thoughts?

Reply:
Will a hero slideshow ever need lazy loading if it's always above the fold?

Reply:
Guess it's more "progressive" than lazy, in that we're loading a low-res placeholder to begin with to make the UI feel fast.

> .u-lazy 

Don't think it's a utility classname as it is, unless we want to change the name to `u-blur` to be more descriptive?

Reply:
Agreed, it is above the fold so ideally shouldn't be called lazy loaded and it is indeed more progressive load.

Blur is more descriptive of the styling effect which gets applied whereas "preloading" is a better descriptor of what's happening technically. In which case how about one of the following?:

`.has-preloador`
`.is-preload`
`.is-preloaded`

Reply:
the `has/is` would work for now, but it restricts it to only the `slideshow__image` selector (`.slideshow__image.is-preload`)

so if we wanted it to be universal selector it should be a utility class imo, and become something (e.g.) `u-img-preload`

Reply:
sold!